src/dafne/utils/sam_mask_refine.py

```diff
@@ -238,10 +240,9 @@ def show_progress(current, maximum):
     bbox = enlarge_bounding_box(mask, GlobalConfig['SAM_BBOX_EXPAND_FACTOR'])
 
     if model_choice in ['Med Sam']:
+        H, W = img.shape[:2]
```

Comment:
I think either `img` or `H` or `W` needs to be set as global variables

Reply:
img is an input parameter, so it works.

Reply:
oh right, makes sense